src/lib.rs

```diff
@@ -159,14 +159,19 @@ pub unsafe extern "C" fn WrapperDecompressImageEx(
 ) -> i32 {
```

Comment:
In the comment above please update the description to "that were compressed by C++ SIMD version of Lepton"